### PR TITLE
No metadata

### PR DESCRIPTION
**Pull request by scopatz**
CC @marcelotrevisani

### CODE REVIEW

File: tests/test_condatools.py [hunk 1]
```diff
@@ -170,3 +170,7 @@ def test_exclude_add_deps(xonsh, data_folder, tmpdir):
         )
         assert "opencv" in wheel.artifact_info.run_requirements
         assert "six" in wheel.artifact_info.run_requirements
+
+
+def test_xeus_python(pip_install_artifact_tree, xonsh):
+    wheel, test_env, sp = pip_install_artifact_tree("xeus-python=0.5.1", skip_python=True, fatten=True)
```
Comment (marcelotrevisani) [inline tests/test_condatools.py]: Should you verify any of these variables? I mean, the content or something like that?

Comment (marcelotrevisani) [reply tests/test_condatools.py]: Or is it a sanity test?

Comment (scopatz) [reply tests/test_condatools.py]: It is more of a sanity test.  Previously, the xeus_python wheel wasn't installable at all.

**Comment by SylvainCorlay** [comment]
Thanks for the heads up @scopatz !

**Comment by SylvainCorlay** [comment]
cc @johanmabille @martinrenou

File: tests/test_condatools.py (hunk 1 above)
Comment (marcelotrevisani) [reply tests/test_condatools.py]: That is fine then! :)

**Pull request merged**